### PR TITLE
docs(readme): update

### PR DESCRIPTION
**Pull request by frosato-dev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,29 +22,48 @@
...
+Cheers,
+
+"His dudeness, duder, or el dudorino"
+
+## Install
+Just run the setup command in your project folder:
 ```sh
-$ npm run dude:format
-$ npm run dude:lint
+$ npm i dudejs
```
Comment (fbarrailla) [inline README.md]:
```
$ npm i dudejs --save-dev
```

File: README.md [hunk 2]
```diff
@@ -22,29 +22,48 @@
...
 ```
+or
+```sh
+$ yarn dudejs
```
Comment (fbarrailla) [inline README.md]:
```
$ yarn add dudejs --dev
```

**Comment by fbarrailla** [comment]
I think that's a good start, well done dude 😄

File: README.md (hunk 1 above)
Comment (frosato-dev) [reply README.md]: Nice catch dude! done :)

File: README.md (hunk 2 above)
Comment (frosato-dev) [reply README.md]: Nice catch dude! done :)

File: README.md [hunk 3]
```diff
@@ -22,29 +22,48 @@
...
+## Install
+Just run the setup command in your project folder:
 ```sh
-$ npm run dude:format
-$ npm run dude:lint
+$ npm i dudejs --save-dev
 ```
+or
+```sh
+$ yarn dudejs --dev
```
Comment (fbarrailla) [inline README.md]: Yarn add ;)

Comment (frosato-dev) [reply README.md]: 😅fixed

**Pull request merged**